.github/workflows/track-acvm.yml

```diff
@@ -0,0 +1,28 @@
+name: Track ACVM for Integration Tests
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
Would be nice to restrict this to when `Cargo.lock` or `yarn.lock` changes. Not sure if we can do an OR on path triggers though (should be able to).

Reply:
Revised